### DISCUSSION TITLE
Pop OS theming quirks

### DISCUSSION THREAD

**Discussion by GiuseppeGalilei** [GradienceTeam/Gradience#707] [General]
I want to start by thanking the community for this wonderful project!

I've been a Pop OS users for the last few years and I've eventually given up theming due to several inconsistencies that would arise in the UI. This project seems to hit the sweet spot.
I've just discovered it and already obtained good results, my experience has been flawless except for the colors in "Pop Shop".
This is due to some quirks in the way Pop and Elementary OS handle theming.

For anyone having the same problem I've found a workaround: https://github.com/jnsh/arc-theme/issues/83#issuecomment-701823965

> You should be able to fix this locally by adding the necessary color definitions in your users' ~/.config/gtk-3.0/gtk.css file. I'm unable to confirm whether this works myself, so I'd be interested to hear the results if anyone tries this.
Here's the list of colors from elementary theme, compiled from https://github.com/elementary/stylesheet/blob/master/src/_palette.scss:

```
@define-color STRAWBERRY_100 #ff8c82;
@define-color STRAWBERRY_300 #ed5353;
@define-color STRAWBERRY_500 #c6262e;
@define-color STRAWBERRY_700 #a10705;
@define-color STRAWBERRY_900 #7a0000;
@define-color ORANGE_100 #ffc27d;
@define-color ORANGE_300 #ffa154;
@define-color ORANGE_500 #f37329;
@define-color ORANGE_700 #cc3b02;
@define-color ORANGE_900 #a62100;
@define-color BANANA_100 #fff394;
@define-color BANANA_300 #ffe16b;
@define-color BANANA_500 #f9c440;
@define-color BANANA_700 #d48e15;
@define-color BANANA_900 #ad5f00;
@define-color LIME_100 #d1ff82;
@define-color LIME_300 #9bdb4d;
@define-color LIME_500 #68b723;
@define-color LIME_700 #3a9104;
@define-color LIME_900 #206b00;
@define-color MINT_100 #89ffdd;
@define-color MINT_300 #43d6b5;
@define-color MINT_500 #28bca3;
@define-color MINT_700 #0e9a83;
@define-color MINT_900 #007367;
@define-color BLUEBERRY_100 #8cd5ff;
@define-color BLUEBERRY_300 #64baff;
@define-color BLUEBERRY_500 #3689e6;
@define-color BLUEBERRY_700 #0d52bf;
@define-color BLUEBERRY_900 #002e99;
@define-color BUBBLEGUM_100 #fe9ab8;
@define-color BUBBLEGUM_300 #f4679d;
@define-color BUBBLEGUM_500 #de3e80;
@define-color BUBBLEGUM_700 #bc245d;
@define-color BUBBLEGUM_900 #910e38;
@define-color GRAPE_100 #e4c6fa;
@define-color GRAPE_300 #cd9ef7;
@define-color GRAPE_500 #a56de2;
@define-color GRAPE_700 #7239b3;
@define-color GRAPE_900 #452981;
@define-color COCOA_100 #a3907c;
@define-color COCOA_300 #8a715e;
@define-color COCOA_500 #715344;
@define-color COCOA_700 #57392d;
@define-color COCOA_900 #3d211b;
@define-color SILVER_100 #fafafa;
@define-color SILVER_300 #d4d4d4;
@define-color SILVER_500 #abacae;
@define-color SILVER_700 #7e8087;
@define-color SILVER_900 #555761;
@define-color SLATE_100 #95a3ab;
@define-color SLATE_300 #667885;
@define-color SLATE_500 #485a6c;
@define-color SLATE_700 #273445;
@define-color SLATE_900 #0e141f;
@define-color BLACK_100 #666;
@define-color BLACK_300 #4d4d4d;
@define-color BLACK_500 #333;
@define-color BLACK_700 #1a1a1a;
@define-color BLACK_900 #000;
```

My current config is Pop Os 22.04 with vanilla gnome 42.

Use this discussion to share possible other problems you find with Pop OS.

**Comment by daudix** [comment]
I think we can add this to preset, thank you for this tip 👍

**Comment by halfmexican** [comment]
OH SICK!! I've stuck to the default theme because of this issue, probably won't switch now that I made the whole pop preset
